### ISSUE TITLE
Upgrade node.js

### ISSUE THREAD

**Issue by oscgonfer** [fablabbcn/smartcitizen-web#445]
Most of the time we are running into issues when maintaining the web. Node v9.8.0 is getting old and bower is starting to complain already:

https://github.com/bower/bower/issues/2623

Would suggest to look into upgrading to a LTS version. 
In the short term, we can't deploy because certificates don't seem to work on `bower`. A solution to test would add the environment variable:

```
NODE_OPTIONS=--use-openssl-ca
```

As per: https://github.com/bower/bower/issues/2608#issuecomment-1521355964

**Comment by oscgonfer** [comment]
**Update**

- Github actions for building with [`setup-node`](https://github.com/actions/setup-node/) the site are seemingly not affected as long as we keep using a version that is available in https://nodejs.org/dist/
- Bower repos will eventually be replaced and probably we need to move away from bower completely to `yarn` or `vite` as explained here: https://bower.io/blog/2017/how-to-migrate-away-from-bower/
- [AngularJS](https://docs.angularjs.org/misc/version-support-status#!) has no support as of January 2022. There is no good solution to this as it was replaced by Angular (in Typescript), and totally rewritten. We should consider if we stay and stretch the situation, or if we move away to a supported framework (a lot to discuss there...)

To summarise: for now, it seems we can get away without more actions but this change will need to happen at some point because the stack seems to be getting old (although c'mon...).

Let's keep this issue open so that we can discuss upgrades for the many items that now are deprecated and see how we move forward.
